GPU/Common/TextureDecoder.h

```diff
@@ -49,7 +49,7 @@ void DoUnswizzleTex16Basic(const u8 *texptr, u32 *ydestp, int bxc, int byc, u32
 #define DoReliableHash32 XXH32
 #define DoReliableHash64 XXH64
 
-#ifdef _M_X64
+#if defined(_M_X64) || defined(ARM64)
```

Comment:
Thanks Unknown! You're correct, of course, however, this wasn't a case of me not being able to read the nested ifdefs. The compile error that an associate reported to me was actually referring to the second change on this PR, and the definition on line 88. I opted to add this as well just as a double check, since there's no explicit conditional in Common.h which ensures _M_SSE isn't defined for ARM64.

Reply:
I mean, if someone explicitly defines _M_SSE on an arm64 device, they deserve the compile errors they get...

-[Unknown]